embulk-input-riak_cs/src/main/java/org/embulk/input/riak_cs/RiakCsFileInputPlugin.java

```diff
@@ -1,11 +1,16 @@
 package org.embulk.input.riak_cs;
 
 import com.amazonaws.ClientConfiguration;
+import com.amazonaws.services.s3.AmazonS3;
 import com.amazonaws.services.s3.AmazonS3Client;
 import org.embulk.config.Config;
-import org.embulk.config.ConfigInject;
 import org.embulk.input.s3.AbstractS3FileInputPlugin;
 
+import static com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration;
+import static com.amazonaws.services.s3.AmazonS3Client.*;
```

Comment:
`import *` should cause `./gradlew checkstyle` failure.
checkstyle plugin isn't installed in this project, though.